invenio_app_ils/records/jsonresolver/circulation.py

```diff
@@ -45,7 +45,7 @@ def _circulation_for_document_resolver(document_pid):
             ),
         )
 
-        circulation["number_of_loans"] = search.execute().hits.total
+        circulation["number_of_past_loans"] = search.execute().hits.total
```

Comment:
I think we should improve this to search for only `ITEM_RETURNED`. WDYT?
Maybe a ticket for this?

Reply:
Does `CANCELLED` count as past loan ?

Reply:
TIcket https://github.com/inveniosoftware/invenio-app-ils/issues/133 :)